lexicons/com/atproto/sync/getRoot.json

```diff
@@ -0,0 +1,34 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.sync.getRoot",
```

Comment:
are we going to switch this to `com.atproto.sync.getLatestCommit` as discussed? I think `getRoot` is ambiguous; "root" also refers to the top node in the MST (vs. the CID of the commit)

Reply:
tried on `getCurrent` for size but then remembered this suggestion. It's a bit wordy for my taste but I think it probably is the most descriptive so I'll probably change it 👍